src/ZTUtils/Zope.py

```diff
@@ -297,7 +303,11 @@ def complex_marshal(pairs):
 def simple_marshal(v):
     if isinstance(v, str):
         return ''
+    if isinstance(v, binary_type):
+        # Py 3 only
+        return ':bytes'
```

Comment:
This relies on the preceding check to return for Python 2 str before this is reached. I can change this to check for PY3 explicitly.